src/main/kotlin/com/ampnet/projectservice/service/impl/CloudStorageServiceImpl.kt

```diff
@@ -67,7 +67,6 @@ class CloudStorageServiceImpl(applicationProperties: ApplicationProperties) : Cl
             logger.info { "Deleted file: $key" }
         } catch (ex: S3Exception) {
             logger.warn { ex.message }
-            throw InternalException(ErrorCode.INT_FILE_STORAGE, "Could not delete file with key: $key on cloud")
```

Comment:
Why avoiding throw?
And if so it must be, then remove @throws in function signature

Reply:
Because delete action on cloud storage is not important for platform functionality.